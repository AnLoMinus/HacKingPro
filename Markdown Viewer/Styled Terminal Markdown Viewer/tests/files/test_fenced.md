```python
class C1(object): pass
```
