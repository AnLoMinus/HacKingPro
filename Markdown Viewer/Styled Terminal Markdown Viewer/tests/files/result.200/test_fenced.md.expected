[38;5;188m  
  [38;5;59m| [0m [38;5;102m[38;5;74mclass[0m[38;5;102m [0m[38;5;102mC1[0m[38;5;102m([0m[38;5;102mobject[0m[38;5;102m)[0m[38;5;102m:[0m[38;5;102m [0m[38;5;74mpass[0m[38;5;102m
[0m[0m


